Fika.Headless/FikaHeadlessPlugin.cs

```diff
@@ -60,7 +63,8 @@ public static bool IsRunningWindows
         private float gcCounter;
         private Coroutine verifyConnectionsRoutine;
         private bool invalidPluginsFound = false;
-
+        private static int currentRaidCount = 0;
+        private static int restartAfterAmountOfRaids = 3;
```

Comment:
Should this not default to 0, and _**if**_ the user decides to use it, the config will override?
```suggestion
        private static int restartAfterAmountOfRaids = 0;
```

Reply:
Done